apps/fhir/bluebutton/fixtures/fhir_bluebutton_new_testdata.json

```diff
@@ -58,7 +58,7 @@
   "fields": {
     "user": 2,
     "fhir_source": 1,
-    "fhir_id": "Patient/9342511",
+    "fhir_id": "Patient/9342512",
```

Comment:
Why this change?

Reply:
oh because the fhir_id's uniqueness is now being enforced at a db level